sphinx/source/schemas/getQuote_result.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Should this and `getQuotes` be added to `users/rpc_api.rst` in the `Common Methods` section?

Reply:
I don't think so, as they're currently only available on the nodes frontend?